save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -191,7 +186,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
     init {
         state.isUploading = false
         state.organization = Organization("", OrganizationStatus.CREATED, null, null, null)
-        state.selectedMenu = OrganizationMenuBar.INFO
+        state.selectedMenu = null
```

Comment:
We definitely should have first initial selection. 
When user opens the window - he should see a particular tab

Reply:
I wanted to load null first, then wait for the url to steam up and load a specific taboo.
I have already replaced it with
```state.selectedMenu = ...MenuBar.default```

Reply:
> I have already replaced it with
> `state.selectedMenu = ...MenuBar.default`

But why is `selectedMenu` still nullable?

Reply:
Seems that now it is non-nullable now. Thanks for reply